### PR TITLE
Android task

### PR DESCRIPTION
**Pull request by ilsen14**

### CODE REVIEW

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java [hunk 1]
```diff
@@ -79,4 +79,25 @@ public CarinaDescriptionPageBase login() {
...
     }
 
+    public boolean isNameInputFieldPresent() {
+        return nameInputField.isPresent();
+    }
+    public boolean isPasswordInputFieldPresent(){
+        return passwordInputField.isPresent();
+    }
+    public boolean isMaleRadioBtnPresent(){
+        return maleRadioBtn.clickIfPresent();
```
Comment (vrudyk) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java]: you should check isElementPresent or isPresent, not click

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java [hunk 2]
```diff
@@ -85,4 +85,29 @@ public void testUIElements() {
...
+    @Test(description = "JIRA#DEMO-0011")
+    @MethodOwner(owner = "qpsdemo")
+    @TestLabel(name = "feature", value = {"mobile", "acceptance"})
+    public void testMyApp(){
+        String username = "Test user";
+        String password = RandomStringUtils.randomAlphabetic(10);
+        WelcomePageBase welcomePage = initPage(getDriver(), WelcomePageBase.class);
+        Assert.assertTrue(welcomePage.isPageOpened(), "Welcome Page is not Opened!");
+        LoginPageBase loginPage = welcomePage.clickNextBtn();
+        Assert.assertTrue(loginPage.isNameInputFieldPresent(), "Name Input Field is not present");
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java]: you should verify isPageOpened
Override method and use it

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java [hunk 3]
```diff
@@ -85,4 +85,29 @@ public void testUIElements() {
...
+        Assert.assertTrue(loginPage.isPasswordInputFieldPresent(), "Password Input Filed is not Present!");
+        Assert.assertTrue(loginPage.isMaleRadioBtnPresent(), "Male Radio Button is not Present");
+        Assert.assertTrue(loginPage.isFemaleRadioBtnPresent(), "Female Radio Button is not Present");
+        Assert.assertTrue(loginPage.isPrivacyPolicyCheckBoxPresent(), "Privacy Policy Field is not Present");
+        loginPage.typeName(username);
+        loginPage.typePassword(password);
```
Comment (vrudyk) [inline src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java:134]: you should verify is text typed or not 
create method for this and use

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/ios/LoginPage.java [hunk 4]
```diff
@@ -15,6 +15,8 @@
 @DeviceType(pageType = Type.IOS_PHONE, parentClass = LoginPageBase.class)
 public class LoginPage extends LoginPageBase {
 
+	private static final String THIS_METHOD_IS_DEFINED_ONLY_IN_ANDROID = "This method is not yet implemented for iOS";
```
Comment (MuslimSolvd) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/ios/LoginPage.java]: A better way I think will be next:
1. create a new class - ProjectConstant
2. define in this class all constants like
public static final String UNIMPLEMENTED_IOS = "Unimplemented for iOS";
public static final String UNIMPLEMENTED_ANDROID = "Unimplemented for Android";
public static final String UNIMPLEMENTED_DESKTOP = "Unimplemented for Desktop";

3. use these constants in the project
4. in the future define another constants for project here

File: src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java [hunk 5]
```diff
@@ -85,4 +85,31 @@ public void testUIElements() {
         Assert.assertTrue(uiElements.isOthersRadioButtonSelected(), "Others radio button was not selected!");
     }
 
+    @Test(description = "JIRA#DEMO-0011")
+    @MethodOwner(owner = "qpsdemo")
```
Comment (MuslimSolvd) [inline src/test/java/com/qaprosoft/carina/demo/MobileSampleTest.java]: You can use value in owner your email - "isenko"

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java [hunk 6]
```diff
@@ -79,4 +79,53 @@ public CarinaDescriptionPageBase login() {
...
+    }
+
+    @Override
+    public boolean isMaleRadioBtnChecked() {
+        return maleRadioBtn.isChecked();
+    }
+
+    @Override
+    public boolean isPrivacyPolicyCheckBoxChecked() {
+        return maleRadioBtn.isChecked();
```
Comment (MuslimSolvd) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/android/LoginPage.java]: I think you mean - return privacyPolicyCheckbox.isChecked();

File: src/main/java/com/qaprosoft/carina/demo/consant/ProjectConstants.java [hunk 7]
```diff
@@ -0,0 +1,11 @@
+package com.qaprosoft.carina.demo.consant;
+
+public  class ProjectConstants {
+
+
```
Comment (MuslimSolvd) [inline src/main/java/com/qaprosoft/carina/demo/consant/ProjectConstants.java]: delete redundant space

File: src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/ios/IOSLoginPage.java [hunk 8]
```diff
@@ -1,5 +1,5 @@
 package com.qaprosoft.carina.demo.mobile.gui.pages.ios;
-
+import com.qaprosoft.carina.demo.consant.ProjectConstants;
 import org.apache.commons.lang3.RandomStringUtils;
 import org.openqa.selenium.WebDriver;
```
Comment (MuslimSolvd) [inline src/main/java/com/qaprosoft/carina/demo/mobile/gui/pages/ios/IOSLoginPage.java]: create import list with this rules:

import java
import org
import com

**Pull request merged**